taskwarrior_syncall/cli.py

```diff
@@ -9,6 +9,46 @@
 from taskwarrior_syncall.constants import COMBINATION_FLAGS
 
 
+def opt_asana_task_gid(**kwargs):
+    return click.option(
+        "-a",
+        "--asana-task",
+        "asana_task_gid",
+        type=str,
+        help="Limit sync to provided task",
+        **kwargs,
+    )
+
+
+def opt_asana_token_pass_path():
+    return click.option(
+        "--token",
+        "--token-pass-path",
+        "token_pass_path",
+        help="Path in the UNIX password manager to fetch",
+        default="app.asana.com/0/my-apps/taskwarrior_syncall",
```

Comment:
Can we remove this default option? Each one ofthe users will have their own password layout so mandating one here wouldn't really work. Also feel free to remove the default from the notion flag [here](https://github.com/bergercookie/taskwarrior-syncall/pull/52/files#diff-eb15d8a04ad5649425284945bae54ca17bb2984ed69827769aed17b454dbf8b7R131)

Reply:
Will do

Reply:
Done in latest commit